.travis.yml

```diff
@@ -1,20 +1,17 @@
 language: php
 
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - 7.1
+
 sudo: false
 
 cache:
     directories:
         - $HOME/.composer/cache/files
 
-matrix:
-    include:
-        - php: 5.5
-        - php: 5.6
-        - php: 7
-    allow_failures:
-        - php: 7
-    fast_finish: true
```

Comment:
Shouldn't this be kept so that the build already reports a failure as soon as one of the PHP version jobs fails?

Reply:
I'd say its unnesseary syntax suggar. I alway wait for all build to finish, to have more data to detect the error. It's easier to find then.

Reply:
The remaining jobs will still finish, but you would be notified earlier (e.g. on GitHub or by mail).

Reply:
I know. My point still remains